src/app/component/ubs-admin/ubs-sidebar/ubs-sidebar.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="sidebar-icons" #sideBarIcons>
+  <ul class="sidebar-list">
+    <li class="sidebar-list-item" (click)="toggleSideBar()">
+      <img class="sidebar-list-item-icon" src="assets\img\sidebarIcons\user_icon.svg">
+    </li>
+    <li class="sidebar-list-item" (click)="toggleSideBar()">
+      <img class="sidebar-list-item-icon" src="assets\img\sidebarIcons\achievment_icon.svg">
+    </li>
+    <li class="sidebar-list-item" (click)="toggleSideBar()">
+      <img class="sidebar-list-item-icon" src="assets\img\sidebarIcons\shopping-cart_icon.svg">
+    </li>
+    <li class="sidebar-list-item" (click)="toggleSideBar()">
+      <img class="sidebar-list-item-icon" src="assets\img\sidebarIcons\workers_icon.svg">
+    </li>
+    <li class="sidebar-list-item" (click)="toggleSideBar()">
+      <img class="sidebar-list-item-icon" src="assets\img\sidebarIcons\documents_icon.svg">
+    </li>
+    <li class="sidebar-list-item" (click)="toggleSideBar()">
+      <img class="sidebar-list-item-icon" src="assets\img\sidebarIcons\calendar_icon.svg">
+    </li>
```

Comment:
can we use angular loop here? so we will have data in array and loop it here instead  of having same code

Reply:
Done